AppCenterAnalytics/AppCenterAnalytics/MSACAnalytics.h

```diff
@@ -197,6 +197,18 @@ NS_SWIFT_NAME(Analytics)
  */
 + (void)resume;
 
+/**
+ Start session manually.
+ *
```

Comment:
please format a comment and remove extra `*`

Reply:
maybe make sense to describe it more in detail as we did in android?
> Start a new session if manual session tracker is enabled, otherwise do nothing.